contracts/leverage_trading/build.sh

```diff
@@ -2,3 +2,5 @@
 set -e
 
 RUSTFLAGS='-C link-arg=-s' cargo build --manifest-path ./Cargo.toml --target wasm32-unknown-unknown --release
+
+cp ./target/wasm32-unknown-unknown/release/leverage_trading.wasm ./res/
```

Comment:
@arsenijkovalov please remove this line as it makes the script
1) unstable in case the folder doesn't exist
2) violates the YAGNI principle because you either can access wasm via both pathes